docs/apm/getting-started.asciidoc

```diff
@@ -8,14 +8,15 @@ the *Setup Instructions* will get you started.
 [role="screenshot"]
 image::apm/images/apm-setup.png[Installation instructions on the APM page in Kibana]
 
-After you install the Elastic APM agent library in your application,
+
+Index patterns tell Kibana which Elasticsearch indices you want to explore.
+An APM index pattern is necessary for certain features in the APM UI.
```

Comment:
Perhaps:
```suggestion
An APM index pattern is necessary for certain features in the APM UI, like the query bar. 
```

(currently the query bar is the only thing that uses index patterns in APM UI)

Reply:
Perfect, thanks for the addition!